adapters/sharethrough/sharethrough.go

```diff
@@ -0,0 +1,72 @@
+package sharethrough
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+const supplyId = "FGMrCMMc"
+const strVersion = "1.0.0"
+
+func NewSharethroughBidder(endpoint string) *SharethroughAdapter {
+	return &SharethroughAdapter{
+		AdServer: StrOpenRTBTranslator{
+			UriHelper: StrUriHelper{BaseURI: endpoint},
+			Util:      Util{},
+		},
+	}
+}
+
+type SharethroughAdapter struct {
+	AdServer StrOpenRTBInterface
+}
+
+func (a SharethroughAdapter) Name() string {
```

Comment:
It is required for the `Adapter` interface, which is only used by the legacy endpoint code. So that is probably why it was created. It also needs the `Call` method, which isn't included, and doesn't support native, so probably best to clean this up.